FirebaseCore/CHANGELOG.md

```diff
@@ -11,6 +11,11 @@
 - Fixed validation issues in Xcode 15.3 that affected binary distributions
   including Analytics, Firestore (SwiftPM binary distribution), and the
   Firebase zip distribution. (#12441)
+- [Zip Distribution] The manual integration instructions found in the
+  `Firebase.zip` have been updated for Xcode 15 users. The updated instructions
```

Comment:
Is there anything different between Xcode 15.3 and earlier Xcode 15 versions?

Reply:
To test, I tried validating an app with all SDKs in Xcode 15.1, and it was successful.

Reply:
I will merge this as is.